ironfish/src/rpc/adapters/tcpAdapter.ts

```diff
@@ -39,6 +40,12 @@ export const IncomingNodeIpcSchema: yup.ObjectSchema<IncomingNodeIpc> = yup
   })
   .required()
 
+type TcpAdapterClient = {
```

Comment:
nit: maybe name Connection instead of Client but either seems fine

Reply:
Isn't the client.socket the connection? What do you generally consider a client? If this only represents a connection, is there no client / user concept?